src/MarketSigner.js

```diff
@@ -402,6 +402,15 @@ function digestBid({
   });
 }
 
+function digestTransferBalanceToWeb3address({
+  owner, toWeb3Address, amount, fromCurrencyId, toCurrencyId, nonce,
```

Comment:
Why is nonce using a uint32? We are using uint256 all across this library

Reply:
for this nonce uint32 is more than enough. There is no way that it can exceed the max value of unit32